src/GuiBase/Utils/KeyMappingManager.hpp

```diff
@@ -45,9 +45,14 @@ class RA_GUIBASE_API KeyMappingManager
     static std::string enumNamesFromKeyboardModifiers( const Qt::KeyboardModifiers& modifiers );
 
     Context getContext( const std::string& contextName ) {
-        return m_contextNameToIndex[contextName];
+        // use find so that it do not insert invalid context
+        auto itr = m_contextNameToIndex.find( contextName );
+        if ( itr != m_contextNameToIndex.end() ) return m_contextNameToIndex[contextName];
+        return Context{};
```

Comment:
Not sure that default context is the better here.
Maybe a specific `ErrorContext` that will capture (and eventually log) all the operations might be more helpful.

Reply:
Default ctor is Invalid index.